app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,13 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".LoginActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
```

Comment:
remove `<action android:name="android.intent.action.MAIN" />` and add `<category android:name="android.intent.category.DEFAULT" />` to `.MainActivity`